### PR TITLE
style: enforce buildifier lint on CI

### PR DESCRIPTION
**Pull request by manekinekko**
## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by manekinekko** [comment]
cc @alexeagle @mgechev 

The reason the build is failing is that the Buildifier had rewritten this line:
```
i18n_messages_files = [ctx.new_file(ctx.genfiles_dir, ctx.label.name + "_ngc_messages.xmb")]
``` 

into this:

```
i18n_messages_files = [ctx.actions.declare_file(ctx.label.name + "_ngc_messages.xmb", sibling = ctx.genfiles_dir)]
```

Because the `ctx.new_file()` is [deprecated](https://github.com/bazelbuild/buildtools/blob/master/WARNINGS.md#ctxaction_name-is-deprecated). However, this triggers an issue that has already been reported in https://github.com/bazelbuild/bazel/issues/4858. So I guess we are blocked by this.

Also, currently, disabling the related warnings isn't implemented (https://github.com/bazelbuild/buildtools/issues/458).

**Comment by manekinekko** [comment]
we are now running the Buildifier with all lint rules except `ctx-actions` and `no-effect`.

**Comment by manekinekko** [comment]
Note: the CI is failing because of this error

```
error https://registry.yarnpkg.com/chalk/-/chalk-2.3.2.tgz: Extracting tar content of undefined failed, the file appears to be corrupt: "Invalid tar header. Maybe the tar is corrupted or it needs to be gunzipped?"
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command
```

**Comment by IgorMinar** [comment]
@manekinekko that error is a flake. yarn/npm have been unhappy in the last few days. can you try rerunning the tests?

**Comment by manekinekko** [comment]
@IgorMinar this should be okay now.

**Review by gkalpak — APPROVED** [review]
LGTM for `elements`, `ngcc`, `service-worker` and `upgrade` :tada:
_**Update:** Once CI is happy_ :grin:

**Comment by IgorMinar** [comment]
CI is not happy :-(

@manekinekko can you please fix the errors? they seem legit.

**Comment by IgorMinar** [comment]
once the PR is green, I can approve for all groups.

**Review by IgorMinar — COMMENTED** [review]
this seems to be heading in the right direction, just needs more polish.

@alexeagle should take a quick look as well.

**Review by alexeagle — COMMENTED** [review]
a couple of files were deleted here, seems like a bad rebase?
I think we should be able to land this tomorrow

File: packages/bazel/src/esm5.bzl [hunk 1]
```diff
@@ -156,8 +156,8 @@ esm5_outputs_aspect = aspect(
             executable = True,
             cfg = "host",
         ),
-        "_tsc_wrapped": attr.label(
-            default = Label("@build_bazel_rules_typescript//:@bazel/typescript/tsc_wrapped"),
+        "_ngc_wrapped": attr.label(
```
[the remark this answers is not in the document]
Comment (manekinekko) [reply packages/bazel/src/esm5.bzl]: my bad, this was a bad rebase. I'm cleaning this.

File: packages/bazel/src/ng_module.bzl [hunk 2]
```diff
@@ -215,7 +215,7 @@ def _expected_outs(ctx):
     # TODO(alxhub): i18n is only produced by the legacy compiler currently. This should be re-enabled
     # when ngtsc can extract messages
     if is_legacy_ngc:
-        i18n_messages_files = [ctx.new_file(ctx.genfiles_dir, ctx.label.name + "_ngc_messages.xmb")]
+        i18n_messages_files = [ctx.actions.declare_file("_ngc_messages.xmb", sibling = ctx.genfiles_dir)]
```
[the remark this answers is not in the document]
Comment (manekinekko) [reply packages/bazel/src/ng_module.bzl]: my bad, this was a bad rebase. I'm cleaning this.

**Comment by manekinekko** [comment]
@IgorMinar @alexeagle @gkalpak I'm closing this PR in favor of https://github.com/angular/angular/pull/28186

I had lots of issues after rebasing.

**Pull request closed without merging**